app/common/constants.ts

```diff
@@ -63,7 +63,7 @@ export enum MainChannels {
 export enum Links {
     DISCORD = 'https://discord.gg/hERDfFZCsH',
     GITHUB = 'https://github.com/onlook-dev/onlook',
-    USAGE_DOCS = 'https://github.com/onlook-dev/onlook?tab=readme-ov-file#using-your-own-react-project',
+    USAGE_DOCS = 'https://github.com/onlook-dev/onlook?tab=readme-ov-file#option-1-use-your-own-react-project',
```

Comment:
Could we use this link instead? https://github.com/onlook-dev/onlook/wiki/How-to-set-up-my-project%3F

Reply:
I have changed the link it to https://github.com/onlook-dev/onlook/wiki/How-to-set-up-my-project%3F